cmd/nexd/main.go

```diff
@@ -148,20 +149,39 @@ func nexdRun(cCtx *cli.Context, logger *zap.Logger, logLevel *zap.AtomicLevel, m
 
 var additionalPlatformFlags []cli.Flag = nil
 
+func NewCustomConfig() zap.Config {
+	// Create a custom encoder configuration
+	encoderConfig := zapcore.EncoderConfig{
+		MessageKey:     "msg",                         // Field name for log message
+		LevelKey:       "level",                       // Field name for log level
+		CallerKey:      "caller",                      // Field name for caller information
+		EncodeLevel:    zapcore.LowercaseLevelEncoder, // Level encoder
+		EncodeCaller:   zapcore.ShortCallerEncoder,    // Caller encoder
+		EncodeDuration: zapcore.StringDurationEncoder,
+	}
+
+	// Create a Zap configuration with custom encoder configuration
+	config := zap.Config{
+		Encoding:         "json",
+		EncoderConfig:    encoderConfig,
+		OutputPaths:      []string{"stdout"},
+		ErrorOutputPaths: []string{"stdout"},
+		Level:            zap.NewAtomicLevelAt(zap.InfoLevel),
+	}
+	return config
+}
 func main() {
 	// set the log level
 	debug := os.Getenv(nexodusLogEnv)
 	var logger *zap.Logger
 	var logLevel *zap.AtomicLevel
 	var err error
 	if debug != "" {
-		logCfg := zap.NewDevelopmentConfig()
-		logLevel = &logCfg.Level
+		logCfg := NewCustomConfig()
 		logger, err = logCfg.Build()
 		logger.Info("Debug logging enabled")
 	} else {
-		logCfg := zap.NewProductionConfig()
-		logLevel = &logCfg.Level
+		logCfg := NewCustomConfig()
```

Comment:
How about still using zap's default production vs. debug configurations, but just setting `logCfg.EncoderConfig.TimeKey = ""` ?

Reply:
Yes, it makes sense. I update the PR.